src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -97,336 +96,151 @@
 	static final MethodHandle TITLE_METHOD;
 
 	static {
+		register(ExprName.class, String.class, "(1¦name[s]|2¦(display|nick|chat|custom)[ ]name[s])", "players/entities/itemtypes/inventories/slots");
+		register(ExprName.class, String.class, "(3¦(player|tab)[ ]list name[s])", "players");

```

Comment:
They are different patterns. I'd argue this is a misuse of parser marks considering you should be doing a pattern check here but parser marks are in some way a tool to avoid pattern-checking code as well. What do the rest think about this? Should the "misuse" if I can even call that be overlooked for sake of code cleanliness?

Reply:
I wasn’t sure how a pattern check here would work since I’m calling register twice. Would the tablist pattern return "2" for matched patterns? I’m not opposed to doing a pattern check at all, I’m just not should if/how it would work since I’m calling register twice.

Reply:
Talked about it in Discord; since Skript doesn't treat the second registration as an additional pattern but rather, a new expression info and this being unhandled behaviour, it's better to leave it as is for now.